addons/web/static/src/js/widgets/date_picker.js

```diff
@@ -77,6 +77,7 @@ var DateWidget = Widget.extend({
         if (this._onScroll) {
             window.removeEventListener('wheel', this._onScroll, true);
         }
+        this.$input.blur(); // force blur before widget is destroyed
```

Comment:
Shouldn't this be done inside the `this.__libInput++; /* ... */ this.__libInput--;` to ensure we don't react to the blur? Or do we want to react to it?

Reply:
it's probably better in `this.__libInput` :+1: 

i think if the datetimepicker still exist, the blur will just hide the date picker, but it's clearer to ensure odoo doesn't react.